<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPUInfoWindow</class>
 <widget class="QWidget" name="GPUInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPU Info</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="busTypeLabel">
     <property name="text">
      <string>Bus Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="currentCoreUseInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="currentCoreClockLabel">
     <property name="text">
      <string>Core Clock</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="currentCoreUseLabel">
     <property name="text">
      <string>GPU Use</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="totalMemoryLabel">
     <property name="text">
      <string>Total Memory</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="cudaCoresLabel">
     <property name="text">
      <string>Cuda Cores</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="busIdInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="busTypeInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="currentCoreTempLabel">
     <property name="text">
      <string>Core Temp</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="currentCoreClockInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="currentCoreTempInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="driverVersionLabel">
     <property name="text">
      <string>Driver Version</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="busIdLabel">
     <property name="text">
      <string>Bus ID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="cudaCoresInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="driverVersionInput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="nameInput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="currentMemoryClockLabel">
     <property name="text">
      <string>Memory Clock</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="currentMemoryClockInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="currentMemoryUseInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="currentMemoryUseLabel">
     <property name="text">
      <string>Memory Use</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="currentFanSpeedLabel">
     <property name="text">
      <string>Fan Speed</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="currentFanSpeedInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="totalMemoryInput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
